charter.md

```diff
@@ -6,7 +6,7 @@ The Interop Team aims to make the web more interoperable in key areas, prioritiz
 
 The team makes decisions based on consensus, which is defined as support from at least two participating organizations and no opposition. Decisions can be made both in the repository and in meetings, as long as consensus can be established.
 
-The chair of the team is responsible for organizing meetings, driving discussions to a conclusion, and recording decisions. The team appoints an individual as the chair based on consensus once per calendar year. There's no term limit; the same individual can be reappointed as chair.
+The chair of the team is responsible for organizing meetings, driving discussions to a conclusion, and recording decisions. The team appoints an individual as the chair based on consensus at a minimum once per calendar year. There's no term limit; the same individual can be reappointed as chair. If the team is unable to meet due to absence of a chair, the Core Team may appoint one directly.
```

Comment:
```suggestion
The chair of the team is responsible for organizing meetings, driving discussions to a conclusion, and recording decisions. The team appoints an individual as the chair based on consensus at a minimum once per calendar year. There's no term limit; the same individual can be reappointed as chair. If the team is unable to meet due to absence of a chair, the [Core Team](https://github.com/orgs/web-platform-tests/teams/wpt-core-team) may appoint one directly.
```

This link only works for those in the core team (and admins and org owners) but it's better than nothing I think.

Reply:
I've filed https://github.com/web-platform-tests/wpt/issues/40212, we can use a better link later if we document the core team.

Reply:
I don't think we should use a link that will point to an error page for the majority of people.